pkg/deploy/generator/resources.go

```diff
@@ -779,6 +779,7 @@ ExecStart=/usr/bin/docker run \
   -e PULL_SECRET \
   -e ADMIN_API_CLIENT_CERT_COMMON_NAME \
   -e RP_MODE \
+  -e ACR_RESOURCE_ID \
```

Comment:
We've got some int values hardcoded in `pkg/env/int.go` and some variables here. Not sure that I understnad the logic behind decisions of what goes where.

Reply:
Here I'm just passing in the enviroment variable that already exists. But the idea here is in the future in prod we might need more than one ACR.